addons/vehicles/CfgVehicles.hpp

```diff
@@ -229,4 +229,27 @@ class CfgVehicles {
             };
         };
     };
+
+    // Wiesel 2
+    class LT_01_base_F : Tank_F {
+        class Turrets : Turrets {
+            class MainTurret : MainTurret {};
+        };
+    };
+    class LT_01_cannon_base_F : LT_01_base_F {
+        class Turrets : Turrets {
+            class MainTurret : MainTurret {};
+        };
+    };
+    class I_LT_01_cannon_F : LT_01_cannon_base_F {
```

Comment:
Can this be done with less repetitive code? I guess I still haven't fully understood config inheritance...

Reply:
@commy2

Reply:
Why not add the weapon to LT_01_cannon_base_F and ignore I_LT_01_cannon_F ?

Reply:
Because it doesn't work. That would have been my first guess.

Reply:
Looking at base configs, it was possible.
I double checked in-game and it's working